pb_plugins/dcsdkgen/utils.py

```diff
@@ -62,6 +62,10 @@ def extract_string_type(field):
     else:
         return "UNKNOWN_TYPE"
 
+def is_primitive_type(field):
+    """ Check if the field type is primitive (e.g. bool,
+    float) or not (e.g message, enum) """
+    return (not field.type in { 11, 14 })
```

Comment:
How do you  get to 11 and 14?

Reply:
https://github.com/grpc/grpc/blob/master/third_party/nanopb/generator/proto/google/protobuf/descriptor.proto#L128